libs/http-wrapper/src/classes/MiddlewareRegistry.js

```diff
@@ -110,7 +110,13 @@ function MiddlewareRegistry() {
      * @param {Object} res
      */
     this.go = function go(req, res) {
-        execute(0, req.method.toLowerCase(), req.url, req, res);
+        try {
+            execute(0, req.method.toLowerCase(), req.url, req, res);
```

Comment:
This will make the middleware stack behave as any other stack implementation out there: it will prevent any uncaught exceptions to crash the server. The user will receive a `500 Internal Server` error.